Gem/CMakeLists.txt

```diff
@@ -10,12 +10,12 @@ set(gem_path ${CMAKE_CURRENT_LIST_DIR})
 set(gem_json ${gem_path}/gem.json)
 o3de_restricted_path(${gem_json} gem_restricted_path gem_parent_relative_path)
 
-# Currently we are in the ROS2-Gem-Demo/Gem folder: ${CMAKE_CURRENT_LIST_DIR}
+# Currently we are in the RobotVacuumSample/Gem folder: ${CMAKE_CURRENT_LIST_DIR}
```

Comment:
Can you add this logic to the top of the this CMakeLists.txt to make it easier to rename this gem in the future?
https://github.com/o3de/o3de/blob/development/Templates/DefaultProject/Template/Gem/CMakeLists.txt#L11-L23

Reply:
Seems to cause an error:
```
CMake Error at Gem/CMakeLists.txt:91 (o3de_find_ancestor_project_root):
  Unknown CMake command "o3de_find_ancestor_project_root".
```

Reply:
Are you using the latest development?
I just checked in the change to support making it easier to rename projects earlier today: https://github.com/o3de/o3de/commit/8ef14617c2fc6ea1a7687c0b2e77ff63497de186

Reply:
thats bleeding edge

Reply:
Is this sample targeting release 2111?

Reply:
no